### ISSUE TITLE
Add social media icons

### ISSUE THREAD

**Issue by nafasebra** [zaydek/heroicons.dev#92]
Hi!
I have a suggest
That was very good if added social media icons to heroicons!

**Comment by zaydek** [comment]
Good idea. Sorry for the late reply. Which icons or how many social media icons do you want to see added?

**Comment by nafasebra** [comment]
> Good idea. Sorry for the late reply. Which icons or how many social media icons do you want to see added?

Trend social media
such as Instagram, Twitter, Facebook, YouTube, Telegram and etc...

**Comment by zaydek** [comment]
@nafasebra I just put out https://feathericons.dev, which features many such icons provided by https://thewolfkit.com (with permission).

I’ll be adding these to Hero shortly but in the meantime you can grab them there! By the way, multi-select is supported so you can grab as many as you need. Let me know if you have any trouble. I’ll keep this open till I roll these changes into Hero.
